composer.json

```diff
@@ -18,6 +18,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "~4.8 || ~5.0",
+        "doctrine/data-fixtures": "^1.1",
         "symfony/monolog-bundle": "~2.4",
         "jackalope/jackalope-doctrine-dbal": "~1.2",
         "zendframework/zendsearch": "2.*@RC"
```

Comment:
but we should replace `zendframework/*` packages this with `handcraftedinthealps/zendsearch: ^2.0`  package